### ISSUE TITLE
Sharetools are not accessible to screen readers

### ISSUE THREAD

**Issue by ChrisBAshton** [bbc/news-vj-sharetools#14]
Our sharetools have `<span>` inside them which is intended for screen readers - but these won't actually get read out to screen readers because they use `display: none`!

See https://github.com/bbc/news-vj-sharetools/blob/eb397bd290c145e4ebf0dd2621b9661717080dea/bin/templates/buttons/share.scss#L14

We should apply the `clip` technique instead, so that the text is hidden to sighted users but read out to screen readers.
